### DISCUSSION TITLE
Add Optional “Slow Leveling” Challenge & Rewards for Players who want to take on a Low Experience Rate (e.g. “Explorer’s Contract”)

### DISCUSSION THREAD

**Discussion by stephenenzor** [chromiecraft/chromiecraft#7642] [Suggestions]
Howdy, first off thank you for all your hard work on this project, I really appreciate it!

I’m someone who likes to level very slow as the journey and quests and role playing is the fun part for me and I want to do as many quests as possible as I approach max level.

I prefer ChromieCraft for a variety of reasons but for reference turtle wow has a neat feature to reward slow playing like this: https://turtle-wow.org/slow-and-steady

Would it be possible to add something similar? I know you can force a lower experience rate but I like the idea a lot of having a banner or some unique rewards for taking on the lower experience rate.

“Slow and steady” as well as the war mode concept (more experience for opting into PvP) seem like great ideas to me.

Anyhow thank you again for your time and have a good one. Apologies if you’ve seen this idea before in some fashion, I didn’t see it with my cursory search.

So, in the same way there’s a “hardcore” mode there could be a bespoke “slow leveling” type mode with unique rewards outside of just forcing it via command line. Thanks again for all the work you do!

**Comment by Yehonal** [comment]
I like this indeed

Something like this? https://v3-azerothshard-org.translate.goog/il-leveling-hardcore/?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=it&_x_tr_pto=wapp

**Reply by stephenenzor** [reply]
Appreciate your comment, will give this a read — thanks.

**Comment by krutoj87** [comment]
I wouldn't mind an additional challenge mode for slow leveling. But the day war mode is added to CC is the day I will stop playing on this server.

**Reply by stephenenzor** [reply]
Isn’t war mode optional on those servers?

**Reply by krutoj87** [reply]
Yes just like p2w stuff on other servers. I still don't want anything like that on CC. There shouldn't be any gameplay advantages tied to stuff like this, just vanity items. Because if you give some players gameplay advantages others don't get, the optional part becomes a lie. Because you'd degrade yourself to a second class player by choosing to play without them.

**Comment by stephenenzor** [comment]
Just an update on other servers they apparently call this the “Explorer’s Contract”. I have updated the title. Thanks,

**Reply by MasterPooBaa** [reply]
Interesting.
I do like that the option " .weekend rate" now works in Chromie for all values between 0 and 1 for normal players. (Even though you get an <Error> message.
Alas it does not carry over to the HardCore challenge though.  I wonder how hard would this be to code/change?

**Comment by kageyfna** [comment]
I like this idea but would also suggest adding a -20% dmg done/ +20% damage taken element. I feel that would make it more classic-like int he level of danger from questing and such
